vendor/assets/stylesheets/fileupload/application.css

```diff
@@ -0,0 +1,3 @@
+/* ...
+*= require_tree .
```

Comment:
Can we use SCSS includes here. That would match how we're doing it in the rest of the gem.